packages/ui/src/views/repo/pull-request/components/pull-request-header.tsx

```diff
@@ -69,10 +69,10 @@ export const PullRequestHeader: React.FC<PullRequestTitleProps> = ({
   return (
     <>
```

Comment:
Is refactoring the only change made in this file?

Reply:
Got it. Change in import path.

```js
import { timeAgo } from '@/utils'
```

Reply:
Yeah, I'll run the linter to fix the rest of the changes👍

Reply:
Linter ignores this file now, so those changes are needed🤷‍♂️